### ISSUE TITLE
Windows Crash v1.10.12 

### ISSUE THREAD

**Issue by SteffenAL** [icing/mod_h2#148]
Happened till now once.

`
[Mon Oct 30 11:03:04.010865 2017] [mpm_winnt:notice] [pid 1516:tid 448] AH00428: Parent: child process 2060 exited with status 3221225477 -- Restarting.`
```
h2_bucket_beam.c

204 {
205    h2_bucket_beam *beam = ctx;
206    pbl->mutex = beam->lock; Unhandled exception thrown: read access violation.**beam** was nullptr. occurred <===
207    pbl->leave = mutex_leave;
208    return apr_thread_mutex_lock(pbl->mutex);
209 }
```

**Call Stack**

```
>	[Inline Frame] mod_http2.so!mutex_enter(void *) Line 206	C	Symbols loaded.
 	[Inline Frame] mod_http2.so!enter_yellow(h2_bucket_beam *) Line 213	C	Symbols loaded.
 	mod_http2.so!h2_beam_get_buffered(h2_bucket_beam * beam) Line 1168	C	Symbols loaded.
 	mod_http2.so!report_stream_iter(void * ctx, void * val) Line 380	C	Symbols loaded.
 	mod_http2.so!ihash_iter(void * ctx, const void * key, int klen, const void * val) Line 270	C	Symbols loaded.
 	libapr-1.dll!74cf6ed1()	Unknown	No symbols loaded.
 	[Frames below may be incorrect and/or missing, no symbols loaded for libapr-1.dll]		Annotated Frame
 	mod_http2.so!h2_ihash_iter(h2_ihash_t * ih, int(*)(void *, void *) fn, void * ctx) Line 279	C	Symbols loaded.
 	mod_http2.so!h2_mplx_release_and_join(h2_mplx * m, apr_thread_cond_t * wait) Line 465	C	Symbols loaded.
 	mod_http2.so!session_cleanup(h2_session * session, const char * trigger) Line 697	C	Symbols loaded.
 	mod_http2.so!h2_session_pre_close(h2_session * session, int async) Line 2279	C	Symbols loaded.
 	mod_http2.so!h2_conn_pre_close(h2_ctx * ctx, conn_rec * c) Line 265	C	Symbols loaded.
 	mod_http2.so!h2_h2_pre_close_conn(conn_rec * c) Line 680	C	Symbols loaded.
 	libhttpd.dll!74d5e9f5()	Unknown	No symbols loaded.
 	mod_http2.so!update_child_status(h2_session * session, int status, const char * msg) Line 1596	C	Symbols loaded.
 	[Inline Frame] mod_http2.so!transit(h2_session *) Line 1621	C	Symbols loaded.
 	[Inline Frame] mod_http2.so!h2_session_ev_local_goaway(h2_session *) Line 1648	C	Symbols loaded.
 	mod_http2.so!dispatch_event(h2_session * session, h2_session_event_t ev, int arg, const char * msg) Line 1927	C	Symbols loaded.
 	mod_http2.so!h2_session_shutdown(h2_session * session, int error, const char * msg, int force_close) Line 671	C	Symbols loaded.
 	[Inline Frame] mod_http2.so!h2_session_ev_conn_error(h2_session *) Line 1676	C	Symbols loaded.
 	mod_http2.so!dispatch_event(h2_session * session, h2_session_event_t ev, int arg, const char * msg) Line 1933	C	Symbols loaded.
 	[External Code]		Annotated Frame
```

**Comment by icing** [comment]
Thanks for the nice stacktrace!

Please try [v1.10.13](https://github.com/icing/mod_h2/releases/tag/v1.10.13).

**Comment by icing** [comment]
Fixed in v1.10.13
